techniques/general/G125.html

```diff
@@ -3,7 +3,7 @@
    </section><section id="description"><h2>Description</h2>
       <p>The objective of this technique is to make it possible for users to locate additional information by providing links to related web pages.
             It is one of a series of techniques for locating content that are sufficient for addressing Success Criterion 2.4.5. 
-            Links are a basic component of the World Wide Web. They are the mechanism that makes the web an interconnected Web of content. Most authors employ this technique automatically when creating web pages.</p>
+            Links are a basic component of the World Wide Web. They are the mechanism that makes the web an interconnected web of content. Most authors employ this technique automatically when creating web pages.</p>
```

Comment:
"...make the web an..."

This feels like it verges on being more like proper noun usage? 
> lowercase except if proper noun meaning World Wide Web